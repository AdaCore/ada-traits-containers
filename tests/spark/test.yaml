description: 'Prove SPARK code'
driver: 'prove'
sources: ['use_sets.adb', 'use_lists.adb', 'use_maps.adb',
          'use_ordered_sets.adb', 'use_vectors.adb']
