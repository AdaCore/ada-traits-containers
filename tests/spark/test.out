conts-cursors.ads:117:13: medium: range check might fail, in instantiation at conts-vectors-generics.ads:278, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:118 (e.g. when Left = Zero)
conts-cursors.ads:117:13: medium: range check might fail, in instantiation at conts-vectors-generics.ads:278, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24 (e.g. when Left = 0)
conts-cursors.ads:117:18: medium: precondition might fail, in instantiation at conts-vectors-generics.ads:278, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:118
conts-cursors.ads:117:18: medium: precondition might fail, in instantiation at conts-vectors-generics.ads:278, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24
conts-cursors.ads:117:21: medium: overflow check might fail, in instantiation at conts-vectors-generics.ads:278, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:118 (e.g. when N = -2147483648)
conts-cursors.ads:117:21: medium: overflow check might fail, in instantiation at conts-vectors-generics.ads:278, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24 (e.g. when N = -2147483648)
conts-cursors.ads:120:13: medium: range check might fail, in instantiation at conts-vectors-generics.ads:278, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:118 (e.g. when Idx = Zero)
conts-cursors.ads:120:13: medium: range check might fail, in instantiation at conts-vectors-generics.ads:278, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24 (e.g. when Idx = 0)
conts-cursors.ads:120:17: medium: precondition might fail, in instantiation at conts-vectors-generics.ads:278, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:118
conts-cursors.ads:120:17: medium: precondition might fail, in instantiation at conts-vectors-generics.ads:278, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24
conts-cursors.ads:135:59: medium: range check might fail, in instantiation at conts-vectors-generics.ads:278, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24
conts-cursors.ads:81:66: medium: range check might fail, in instantiation at conts-cursors.ads:134, in instantiation at conts-vectors-generics.ads:278, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24
conts-functional-sequences.ads:65:55: medium: range check might fail, in instantiation at conts-vectors-impl.ads:156, in instantiation at conts-vectors-generics.ads:50, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24
conts-lists-generics.ads:168:28: medium: precondition might fail, in instantiation at conts-lists-indefinite_unbounded_spark.ads:47, in instantiation at use_lists.ads:9
conts-lists-generics.ads:170:28: medium: precondition might fail, in instantiation at conts-lists-indefinite_unbounded_spark.ads:47, in instantiation at use_lists.ads:9
conts-lists-generics.ads:181:24: medium: precondition might fail, in instantiation at conts-lists-indefinite_unbounded_spark.ads:47, in instantiation at use_lists.ads:9
conts-lists-generics.ads:184:66: medium: precondition might fail, in instantiation at conts-lists-indefinite_unbounded_spark.ads:47, in instantiation at use_lists.ads:9
conts-lists-impl.ads:393:10: medium: precondition might fail, in instantiation at conts-lists-generics.ads:53, in instantiation at conts-lists-indefinite_unbounded_spark.ads:47, in instantiation at use_lists.ads:9 (e.g. when Self = 1)
conts-lists-impl.ads:398:11: medium: precondition might fail, in instantiation at conts-lists-generics.ads:53, in instantiation at conts-lists-indefinite_unbounded_spark.ads:47, in instantiation at use_lists.ads:9
conts-lists-impl.ads:398:34: medium: precondition might fail, in instantiation at conts-lists-generics.ads:53, in instantiation at conts-lists-indefinite_unbounded_spark.ads:47, in instantiation at use_lists.ads:9 (e.g. when Self = 1)
conts-lists-impl.ads:416:10: medium: precondition might fail, in instantiation at conts-lists-generics.ads:53, in instantiation at conts-lists-indefinite_unbounded_spark.ads:47, in instantiation at use_lists.ads:9 (e.g. when Self = 1)
conts-maps-generics.ads:203:28: medium: precondition might fail, in instantiation at conts-maps-indef_indef_unbounded_spark.ads:49, in instantiation at use_maps.ads:11
conts-maps-generics.ads:213:44: medium: precondition might fail, in instantiation at conts-maps-indef_indef_unbounded_spark.ads:49, in instantiation at use_maps.ads:11
conts-maps-generics.ads:215:48: medium: precondition might fail, in instantiation at conts-maps-indef_indef_unbounded_spark.ads:49, in instantiation at use_maps.ads:11
conts-maps-generics.ads:218:29: medium: precondition might fail, in instantiation at conts-maps-indef_indef_unbounded_spark.ads:49, in instantiation at use_maps.ads:11
conts-maps-generics.ads:221:29: medium: precondition might fail, in instantiation at conts-maps-indef_indef_unbounded_spark.ads:49, in instantiation at use_maps.ads:11
conts-maps-impl.ads:433:10: medium: precondition might fail, in instantiation at conts-maps-generics.ads:74, in instantiation at conts-maps-indef_indef_unbounded_spark.ads:49, in instantiation at use_maps.ads:11 (e.g. when Self = 1)
conts-maps-impl.ads:448:10: medium: precondition might fail, in instantiation at conts-maps-generics.ads:74, in instantiation at conts-maps-indef_indef_unbounded_spark.ads:49, in instantiation at use_maps.ads:11 (e.g. when Self = 1)
conts-vectors-generics.ads:231:09: medium: default initial condition might fail, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24
conts-vectors-generics.ads:264:23: medium: range check might fail, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24
conts-vectors-generics.ads:281:32: medium: range check might fail, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24
conts-vectors-generics.ads:292:28: medium: precondition might fail, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:118
conts-vectors-generics.ads:292:28: medium: precondition might fail, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24
conts-vectors-generics.ads:293:28: medium: precondition might fail, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:118
conts-vectors-generics.ads:293:28: medium: precondition might fail, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24
conts-vectors-generics.ads:304:24: medium: precondition might fail, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:118
conts-vectors-generics.ads:304:24: medium: precondition might fail, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24
conts-vectors-generics.ads:307:68: medium: precondition might fail, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:118
conts-vectors-generics.ads:307:68: medium: precondition might fail, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24
conts-vectors-generics.ads:310:24: medium: precondition might fail, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:118
conts-vectors-generics.ads:310:24: medium: precondition might fail, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24
conts-vectors-generics.ads:310:24: medium: range check might fail, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:118
conts-vectors-generics.ads:310:24: medium: range check might fail, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24
conts-vectors-generics.ads:56:48: medium: range check might fail, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24 (e.g. when No_Index = 0)
conts-vectors-impl.ads:48:56: medium: range check might fail, in instantiation at conts-vectors-generics.ads:50, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24 (e.g. when No_Index = 0)
conts-vectors-impl.ads:528:10: medium: precondition might fail, in instantiation at conts-vectors-generics.ads:50, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:118 (e.g. when Self = 1)
conts-vectors-impl.ads:528:10: medium: precondition might fail, in instantiation at conts-vectors-generics.ads:50, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24 (e.g. when Self = 1)
conts-vectors-impl.ads:533:11: medium: precondition might fail, in instantiation at conts-vectors-generics.ads:50, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:118
conts-vectors-impl.ads:533:11: medium: precondition might fail, in instantiation at conts-vectors-generics.ads:50, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24
conts-vectors-impl.ads:533:34: medium: precondition might fail, in instantiation at conts-vectors-generics.ads:50, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:118
conts-vectors-impl.ads:533:34: medium: precondition might fail, in instantiation at conts-vectors-generics.ads:50, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24
conts-vectors-impl.ads:551:10: medium: precondition might fail, in instantiation at conts-vectors-generics.ads:50, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:118 (e.g. when Self = 1)
conts-vectors-impl.ads:551:10: medium: precondition might fail, in instantiation at conts-vectors-generics.ads:50, in instantiation at conts-vectors-indefinite_unbounded_spark.ads:43, in instantiation at use_vectors.ads:24 (e.g. when Self = 1)